### ISSUE TITLE
Save clears breakpoints

### ISSUE THREAD

**Issue by spazmodius** [ideaconnect/vs-trim-line-ends-on-save-plugin#4]
Maybe the extension is touching every line, whether it needs trimming or not?

**Comment by spazmodius** [comment]
I also lose my bookmarks

**Comment by AndrewJD79** [comment]
Bookmarks clearing makes the extension unusable :(

**Comment by fnxweb** [comment]
Yes, it reads the whole file/buffer into a string, does a search and replace on it, and replaces the entire buffer again.  I had a look a while back to see if there was a way to manually walk through the buffer (there must be), but I didn't find it and ran out of time.
